src/en/get-involved.md

```diff
@@ -50,42 +56,44 @@ Provide any of the following for each component or pattern:
 - **Code**: prototype or production code, or product implementations.
 
 <gcds-link href="{{ links.githubCompsPriority }}" external>Contribute to next priorities</gcds-link>
+Provide any of the following for each component or pattern:
+- **Examples of designs**: screenshots, prototypes, links to live services.
+- **Research findings**: your discovery, use cases, or usability testing
+- **Code**: prototype or production code, or product implementations.
+
+<gcds-grid class="mb-400 hydrated" columns-tablet="1fr 1fr">
+<gcds-button href="{{ links.githubCompsPriority }}" external>Contribuer sur GitHub</gcds-button>
+<gcds-button href="{{ links."/fr/contactez" }}" external>Contactez-nous</gcds-button>
 
+</gcds-grid>
```

Comment:
- Buttons should not need a grid component to render side by side.
- Also there is a typo in the links data that is stopping the site from building. 
- Changed buttons to secondary buttons since we shouldn't have two primary buttons on the same page.
- Link text is in French

```suggestion
<gcds-button button-role="secondary" button-type="link" href="{{ links.githubCompsPriority }}" external>Contribuer sur GitHub</gcds-button>
<gcds-button button-role="secondary" button-type="link" href="{{ links.contact }}" external>Contactez-nous</gcds-button>
```